### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by pollyndos**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,29 +1,66 @@
...
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
+# Dashboard for WinForms - How to Pass a Hidden Dashboard Parameter to a Custom SQL query
 
-* [Form1.cs](./CS/Dashboard_CustomParameters_Win/Form1.cs) (VB: [Form1.vb](./VB/Dashboard_CustomParameters_Win/Form1.vb))
-<!-- default file list end -->
-# Dashboard for WinForms - Pass a Hidden Dashboard Parameter to a Custom SQL query
+This example shows how to pass a hidden [dashboard parameter](https://docs.devexpress.com/Dashboard/16169) to a [custom SQL query](https://docs.devexpress.com/Dashboard/115212). In this example, the[`DashboardViewer.CustomParameters`](https://docs.devexpress.com/Dashboard/DevExpress.DashboardWin.DashboardViewer.CustomParameters) is handled to change the dashboard parameter's default value before it is passed to the query.
```
Comment (albertov05) [inline Readme.md]:
```suggestion
This example shows how to pass a hidden [dashboard parameter](https://docs.devexpress.com/Dashboard/16169) to a [custom SQL query](https://docs.devexpress.com/Dashboard/115212). In this example, the [`DashboardViewer.CustomParameters`](https://docs.devexpress.com/Dashboard/DevExpress.DashboardWin.DashboardViewer.CustomParameters) event is handled to change the dashboard parameter's default value before it is passed to the query. 
```

File: Readme.md [hunk 2]
```diff
@@ -1,29 +1,66 @@
...
+
+To pass a hidden dashboard parameter to a custom SQL query, do the following.
+
+### Create a Dashboard Parameter
+
+In the [Dashboard Designer](https://docs.devexpress.com/Dashboard/117006/), create a [dashboard parameter](https://docs.devexpress.com/Dashboard/16169/). 
+
+Click the **Parameters** button from the **Dashboard** page group on the **Ribbon**'s **Home** page. Click **Add** to create a dashboard parameter. Diasble the [Visible](https://docs.devexpress.com/Dashboard/DevExpress.DashboardCommon.DashboardParameter.Visible) property to hide the parameter from the **Dashboard Parameters** dialog. In this example, the dashboard parameter's name is **custIDDashboardParameter** and its default value is *ALFKI*:
+
+![Dashbaord Parameter Settings](images/dashboard-parameter-settings.png)
```
Comment (albertov05) [inline Readme.md]:
```suggestion
![Dashboard Parameter Settings](images/dashboard-parameter-settings.png)
```

File: Readme.md [hunk 3]
```diff
@@ -1,29 +1,66 @@
...
+
+### Create a Custom Query
+
+Custom SQL queries are disabled by default. To allow users to specify custom SQL queries in the [Query Builder](https://docs.devexpress.com/Dashboard/16152)/[Query Editor](https://docs.devexpress.com/Dashboard/115206), set the [`SqlWizardSettings.EnableCustomSql`](https://docs.devexpress.com/Dashboard/DevExpress.DataAccess.UI.Wizard.SqlWizardSettings.EnableCustomSql) property exposed by the [`DashboardDataSourceWizardSettings`](https://docs.devexpress.com/Dashboard/DevExpress.DashboardWin.DashboardDataSourceWizardSettings) class to `true`. Add the following code to allow custom SQL queries:
+ 
+```csharp
+dashboardDesigner1.DataSourceWizard.SqlWizardSettings.EnableCustomSql = true;
+```
+
+Specify the query in the **SQL string** editor in [Query Builder](https://docs.devexpress.com/Dashboard/16152)/[Query Editor](https://docs.devexpress.com/Dashboard/115206):
```
Comment (albertov05) [inline Readme.md]:
```suggestion
Specify the query in the **SQL string** editor in the [Query Builder](https://docs.devexpress.com/Dashboard/16152)/[Query Editor](https://docs.devexpress.com/Dashboard/115206):
```

File: Readme.md [hunk 4]
```diff
@@ -1,29 +1,66 @@
...
 
+### Bind the Query Parameter to the Dashboard Parameter
 
-The **custIDQueryParameter** query parameter is included in a **WHERE** clause of a [custom SQL query](https://docs.devexpress.com/Dashboard/115212). The **custIDQueryParameter** parameter is also bound to the hidden **custIDDashboardParameter** dashboard parameter. The value of this parameter is changed at runtime by handling the [DashboardViewer.CustomParameters](https://docs.devexpress.com/Dashboard/DevExpress.DashboardWin.DashboardViewer.CustomParameters) event that is raised before the DashboardViewer queries a database.
+To be able to change the query parameter's value, bind it to the **custIDDashboardParameter** dashboard parameter. To do this, on the next page of the **Query Editor** click **Add** to create a query parameter. Specify the parameter [settings](https://docs.devexpress.com/Dashboard/16169/). In the **Name** column, specify the name used in the query. Select the **Expression** checkbox and bind the query parameter to the dashboard parameter:
```
Comment (albertov05) [inline Readme.md]: 1. `To be able to change the query parameter's value`...

Can you clarify why you say "To **be able to** change the..." here? If you're explaining how to change the query parameter value, then did you mean to say "To change the..."?

2. `To do this, on the next page of the **Query Editor** click **Add** to create a query parameter.`

Please add a comma after "Query Editor".

3. `Select the **Expression** checkbox and bind the query parameter to the dashboard parameter:`

Let's change "checkbox" to "check box".

File: Readme.md [hunk 5]
```diff
@@ -1,29 +1,66 @@
...
+
+Click **Finish**.
+
+### Change the Default Parameter Value in Code (Dashboard Viewer)
+
+[Load](https://docs.devexpress.com/Dashboard/16913/) the created dashboard to the [Dashboard Viewer](https://docs.devexpress.com/Dashboard/117122/).
+
+Handle the [`DashboardViewer.CustomParameters`](https://docs.devexpress.com/Dashboard/DevExpress.DashboardWin.DashboardViewer.CustomParameters) event and specify the value to be passed to the query: [Form1.cs](./CS/Dashboard_CustomParameters_Win/Form1.cs) (VB: [Form1.vb](./VB/Dashboard_CustomParameters_Win/Form1.vb)).
+
+As the result, a user sees a dashboard based on the data from the SQL query with the **custIDQueryParameter** query parameter's value specified in the `DashboardViewer.CustomParameters` event handler (*AROUT*).
```
Comment (albertov05) [inline Readme.md]:
```suggestion
As a result, a user sees a dashboard based on the data from the SQL query with the **custIDQueryParameter** query parameter's value specified in the `DashboardViewer.CustomParameters` event handler (*AROUT*).
```

File: Readme.md (hunk 4 above)
Comment (pollyndos) [reply Readme.md]: 1. The idea is to change the query parameter value dynamically. A user can set a static value or bind it to the dashboard parameter.

How about this:

```suggestion
To change the query parameter value dynamically, bind it to the **custIDDashboardParameter** dashboard parameter. To do this, on the next page of the **Query Editor**, click **Add** to create a query parameter. Specify the parameter [settings](https://docs.devexpress.com/Dashboard/16169/). In the **Name** column, specify the name used in the query. Select the **Expression** check box and bind the query parameter to the dashboard parameter:
```

Comment (albertov05) [reply Readme.md]: Sounds good. Thank you!

**Pull request merged**